public/silent-callback.html

```diff
@@ -0,0 +1,8 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/oidc-client/1.10.1/oidc-client.min.js"></script>
+
+<script>
+  var mgr = new Oidc.UserManager();
+  mgr.signinSilentCallback().catch(error => {
+    console.error('silent-callback.html error', error);
+  });
+</script>
```

Comment:
Would it be better to do in the app itself and with redux-oidc?

see:
https://github.com/maxmantz/redux-oidc/blob/master/docs/API.md#processsilentrenew
https://github.com/City-of-Helsinki/palvelutarjotin-admin/blob/develop/src/domain/auth/silentRenew/SilentRenew.tsx

Reply:
My initial idea was that the load time of the independent silent-callback.html would be smaller. But stupid idea because the JS files are coming from the browser cache anyway.

Changed

Reply:
Hmm I don't actually know which way is more performant. Silent callback page is loaded within iframe in the app? BTW what is the cache time for JS files?

Reply:
Anyway I think that it is little bit clearer to have it included as a route in the app. Took me little bit to understand how silent callback was implemented because the page was in a static file and done differently compared to palvelutarjotin

Reply:
Maybe 20ms but I think it's not even very important. It happens only once in an hour.